src/p2p/peer_comm_session.cpp

```diff
@@ -13,10 +13,7 @@ namespace p2pmsg = msg::fbuf::p2pmsg;
 
 namespace p2p
 {
-    // The set of recent peer message hashes used for duplicate detection.
-    util::rollover_hashset recent_peermsg_hashes(200);
-
-    /**
+     /**
```

Comment:
where's the actual use of the bloom filter to deduplicate? you probably need to run two at the same time so that periodically you can clear one or the other to prevent them filling up and becoming useless

Reply:
The bloom filter implementation maintains the exact same interface (try_emplace) so no other code changes are needed. The global recent_peermsg_hashes is now defined in bloom_filter.hpp and works exactly the same way.

Will look at running two bloom filters at the same time and will amend